deployment/README.md

```diff
@@ -1,11 +1,10 @@
 # Human-Connection Nitro \| Deployment Configuration
 
-We deploy with [kubernetes](https://kubernetes.io/). In order to deploy your own
-network you have to [install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-and get a kubernetes cluster.
+There are a couple different ways we have tested to deploy an instance of Human Connection, with [kubernetes](https://kubernetes.io/) and via [Helm](https://helm.sh/docs/). In order to manage your own
```

Comment:
I would even argue that we do not support plain kubernetes configurations once we're familiar enough with helm.

Reply:
yeah, I hear ya! One more dependency that needs to be updated/tested